main.py

```diff
@@ -1,10 +1,45 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players = json.load(file)
+    for player_name, player_data in players.items():
+        race_data = player_data["race"]
+        race, created = Race.objects.get_or_create(
```

Comment:
You could omit the second returned argument since it is not used
```suggestion
        race, _ = Race.objects.get_or_create(
```